### ISSUE TITLE
About Task-adaptive Fine-tuning Inference(TFI)

### ISSUE THREAD

**Issue by King-king424** [slei109/PATNet#24]
How do I use TFI before I get the final prediction mask?
I hope you could provide the relevant code. 
Thanks!

**Comment by slei109** [comment]
Thanks for your interests on our work. We provide the key code named finetune_reference() in the patnet.py.
